tests/behat/features/Paragraphs/WYSIWYG.feature

```diff
@@ -21,6 +21,7 @@ Feature: WYSIWYG Paragraph
     And I wait 1 seconds
     Then I fill in wysiwyg "Body" with "../../assets/documents/WYSIWYG.html"
     Then I press "Continue"
```

Comment:
Not within the scope of this PR, but why are these "Press" and not "Click"?

Reply:
button = "press" vs link = "click"

Reply:
Right. Then why not the `'Given I press the "<name>" button'` step definition from the Behat Drupal Extension? This isn't custom, is it?

Reply:
its not custom. i think its just shorthand for the same step

Reply:
Interdasting.

Reply:
i do wonder if `Then I press the "Continue" button` would've fixed the failures. :man_shrugging: